migrations/2022-08-20-151615_create_fang_tasks/up.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE fang_tasks (
      task_type VARCHAR DEFAULT 'common' NOT NULL,
      periodic BOOLEAN DEFAULT FALSE,
      uniq_hash CHAR(64),
+     period_in_millis BIGINT,
```

Comment:
I think we don't need this.

Let's use cron to schedule the next execution

My idea:

add an additional method into Runnable trait
this mehtod will return Option<cron schedule>.

Reply:
[delete period in millis](https://github.com/ayrat555/fang/pull/71/commits/726ea1bd87f740c3f0cc60e6c92c428aa47891ad)

Reply:
[delete period in millis completed](https://github.com/ayrat555/fang/pull/71/commits/2fa340831d07b3791761815bde2e68b827774f3b)

Reply:
[Cron and single Schedule support :D](https://github.com/ayrat555/fang/pull/71/commits/e48bc9b78f37fd83210476bc7fc0df4109672dc6)